src/test/java/org/testcontainers/containers/TarantoolContainerIT.java

```diff
@@ -73,31 +74,14 @@ public void testContainerWithParameters() throws Exception {
 
     @Test
     public void testContainerWithTrueVersion() throws Exception {
-        final String version = "2.11.0";
-        addEnv(ENV_TARANTOOL_VERSION, version);
-
-        List<String> result;
-        try (TarantoolContainer container = new TarantoolContainer()) {
-            container.start();
-            result = container.executeCommandDecoded("return _TARANTOOL");
-        }
-
-        removeEnv(ENV_TARANTOOL_VERSION, version);
-        assertEquals(1, result.size());
-        assertTrue(result.get(0).startsWith(version));
-    }
-
-    @Test
-    public void testContainerWithDefaultVersionVersion() throws Exception {
-
         List<String> result;
         try (TarantoolContainer container = new TarantoolContainer()) {
             container.start();
             result = container.executeCommandDecoded("return _TARANTOOL");
         }
 
         assertEquals(1, result.size());
-        assertTrue(result.get(0).startsWith(TarantoolContainer.DEFAULT_IMAGE_VERSION));
+        assertTrue(result.get(0).startsWith(String.valueOf(tarantoolVersion.charAt(0))));
```

Comment:
What if these are not CI tests with predefined Tarantool versions, and there is no TARANTOOL_VERSION variable on the system on which the test runs? It seems that a NullPointerException is possible

Reply:
Added if statement